src/app/mods/grip-position-save/grip-position-save.ts

```diff
@@ -0,0 +1,182 @@
+// TODO :
+// - Use ipcRendererService instead of wGame.addEventListener('storage', ...
+// Can't make ipcRendererService works : this.ipcRendererService is well instantiate
+// If you log this.ipcRendererService you see in Listener on move-grip
+// But this.ipcRendererService.send('move-grip', target) doesn't ùake anything
+
+import { SettingsService } from "@services/settings.service";
+import { TranslateService } from "@ngx-translate/core";
+
+import { Mod } from "../mod";
+import { IpcRendererService } from "@services/electron/ipcrenderer.service";
+
+export class GripPositionSave extends Mod {
+
+  constructor(
+    wGame: any,
+    settings: SettingsService,
+    translate: TranslateService,
+    public ipcRendererService: IpcRendererService
+  ) {
+    super(wGame, settings, translate);
+    this.ipcRendererService = ipcRendererService;
+
+    this.registerEvent(this.wGame.gui.isConnected);
+
+    // This is a temporary fix because ipcRendererService doesn't work
+    // It works only for accounts in same Lindo window, not in multiple Lindo
+    // E.G :
+    // - You have 2 windows (A, B) of 4 accounts (1, 2, 3 ...)
+    // - You move Party in Window A, Account 1
+    // - Party is moved on every accounts logged in Window A, not in Window B
+    wGame.addEventListener('storage', (e) => {
+      const target = e.key.substr(0, e.key.length - 8);
+      this.moveElement(target);
+    });
+  }
+
+  startMod(): void {
+    return;
+  }
+
+  public registerEvent(skipLogin: boolean = false): void {
+    const onCharacterSelectedSuccess = () => {
+
+      // This doesn't work, the event is never fired
+      this.on(this.ipcRendererService, 'move-grip', (event: Event, target: any) => {
+        // Never displayed in console, this.moveElement never called.
+        console.log('Got event! ' + target);
```

Comment:
Remove the code if not called ?

Reply:
Nop, because I **should** use ipcRendererService, but it doesn't work and I don't know why.
I've never used ipcRendererService.
I copied the logic from auto-group mod, but without success.